x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -286,11 +286,31 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
   useEffect(() => {
     // Row classes do not deal with visible row indices, so we need to handle page offset
     const rowIndex = flyoutAlertIndex + pagination.pageIndex * pagination.pageSize;
-    setRowClasses({
-      [rowIndex]: ACTIVE_ROW_CLASS,
+    setActiveRowClasses({
+      [rowIndex]: 'alertsTableActiveRow',
     });
   }, [flyoutAlertIndex, pagination.pageIndex, pagination.pageSize]);
 
+  // Update highlighted rows when alerts or pagination changes
+  const highlightedRowClasses = useMemo(() => {
+    let mappedRowClasses: EuiDataGridStyle['rowClasses'] = {};
+    const highlightedRowMapper = props.highlightedRowMapper;
+    if (highlightedRowMapper) {
+      mappedRowClasses = alerts.reduce<NonNullable<EuiDataGridStyle['rowClasses']>>(
+        (rowClasses, alert, index) => {
+          const shouldHighlightRow = highlightedRowMapper(alert);
```

Comment:
`nit`: By name `highlightedRowMapper` seems that it is mapping the `rowclasses` ( what you were doing previously). Could we rename to something like `shouldHighlightRow`?

Reply:
Great idea 👍 https://github.com/elastic/kibana/pull/155497/commits/568b1ffdcbb11d08f2733ddf9ac05e263cbe668f